package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "output",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
And package.lock...

Reply:
yes am doing it from terminal but it added so many packages.. im wondering if it's correct or not

Reply:
in the current package-lock we only have `dependencies: {...}`, running npm i would add `packages: {...}`

Reply:
Expected